ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -60,11 +61,21 @@ export const AnacondaWizard = ({ dispatch, isBootIso, osRelease, storageData, lo
     const [storageEncryption, setStorageEncryption] = useState(getStorageEncryptionState());
     const [storageScenarioId, setStorageScenarioId] = useState(window.sessionStorage.getItem("storage-scenario-id") || getDefaultScenario().id);
     const [reusePartitioning, setReusePartitioning] = useState(false);
+    const [requiredMountPoints, setRequiredMountPoints] = useState();
 
     const availableDevices = useMemo(() => {
         return Object.keys(storageData.devices);
     }, [storageData.devices]);
 
+    useEffect(() => {
+        const updateRequiredMountPoints = async () => {
+            const requiredMountPoints = await getRequiredMountPoints().catch(console.error);
+
+            setRequiredMountPoints(requiredMountPoints);
+        };
+        updateRequiredMountPoints();
+    }, []);
```

Comment:
I actually wonder, can't you do something like this?
```
    useEffect(() => {
            getRequiredMountPoints()
                .then(required_mount_points => setRequiredMountPoints(required_mount_points))
                .catch(console.error);
    }, []);
```

Reply:
That's possible. It's just another style of writing this.